CHANGELOG.md

```diff
@@ -2,12 +2,60 @@
 
 ## Table of Contents
 
+- [r2.1](#r21)
 - **[r1.2](#r12)**
 - [r1.1](#r11)
 - [v0.1.0](#v010)
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# r2.1
+## Release Notes
+
+This release contains the definition and documentation of
+* population-density-data v0.2.0-rc.1
+
+The API definition(s) are based on
+* Commonalities v0.5.0-alpha.1
+* Identity and Consent Management v0.3.0-alpha.1
```

Comment:
I have the impression that you are aligned with the release candidate versions. Please double check with https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/62357505/Analysis+of+Commonalities+0.5.0-rc.1+changes and change here if all is done:

```suggestion
* Commonalities v0.5.0-rc.1
* Identity and Consent Management v0.3.0-rc.1
```